apps/code/script_store.h

```diff
@@ -1,36 +1,36 @@
 #ifndef CODE_SCRIPT_STORE_H
 #define CODE_SCRIPT_STORE_H
 
-#include "accordion.h"
 #include "script.h"
 #include "script_template.h"
 #include <python/port/port.h>
 extern "C" {
 #include "py/parse.h"
 }
 
+extern Ion::Storage storage;
+
 namespace Code {
 
 class ScriptStore : public MicroPython::ScriptProvider {
 public:
-  enum class EditableZone {
-    None,
-    Name,
-    Content
-  };
-
   static constexpr char k_scriptExtension[] = ".py";
   static constexpr char k_defaultScriptName[] = "script.py";
```

Comment:
Wouldn't a better name for an untitled script be `untitled.py`, like what most text editors on computers would do?

Reply:
Good idea but we should localized 'untitled'. It is worth a new pull request!